<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsPage</class>
 <widget class="QWidget" name="SettingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>741</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTabWidget" name="tabs">
     <property name="currentIndex">
      <number>3</number>
     </property>
     <property name="documentMode">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="tabUserInterface">
      <attribute name="title">
       <string/>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <property name="horizontalSpacing">
        <number>8</number>
       </property>
       <property name="verticalSpacing">
        <number>8</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="lblTheme"/>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="cmbTheme"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lblUpdates"/>
       </item>
       <item row="1" column="1">
        <widget class="QCheckBox" name="chkUpdates"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lblCompletedNotificationTrigger"/>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="cmbCompletedNotificationTrigger"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lblPreventSuspend"/>
       </item>
       <item row="3" column="1">
        <widget class="QCheckBox" name="chkPreventSuspend"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="lblRecoverCrashedDownloads"/>
       </item>
       <item row="4" column="1">
        <widget class="QCheckBox" name="chkRecoverCrashedDownloads"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="lblDownloadImmediately"/>
       </item>
       <item row="5" column="1">
        <widget class="QCheckBox" name="chkDownloadImmediately"/>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="lblHistoryLength"/>
       </item>
       <item row="6" column="1">
        <widget class="QComboBox" name="cmbHistoryLength"/>
       </item>
       <item row="7" column="1">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabDownloads">
      <attribute name="title">
       <string/>
      </attribute>
      <layout class="QFormLayout" name="formLayout_2">
       <property name="horizontalSpacing">
        <number>8</number>
       </property>
       <property name="verticalSpacing">
        <number>8</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="lblMaxNumberOfActiveDownloads"/>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="cmbMaxNumberOfActiveDownloads"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lblOverwriteExistingFiles"/>
       </item>
       <item row="1" column="1">
        <widget class="QCheckBox" name="chkOverwriteExistingFiles"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lblLimitCharacters"/>
       </item>
       <item row="2" column="1">
        <widget class="QCheckBox" name="chkLimitCharacters"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lblIncludeMediaId"/>
       </item>
       <item row="3" column="1">
        <widget class="QCheckBox" name="chkIncludeMediaId"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="lblIncludeAutoGeneratedSubtitles"/>
       </item>
       <item row="4" column="1">
        <widget class="QCheckBox" name="chkIncludeAutoGeneratedSubtitles"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="lblPreferredVideoCodec"/>
       </item>
       <item row="5" column="1">
        <widget class="QComboBox" name="cmbPreferredVideoCodec"/>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="lblPreferredSubtitleFormat"/>
       </item>
       <item row="6" column="1">
        <widget class="QComboBox" name="cmbPreferredSubtitleFormat"/>
       </item>
       <item row="7" column="1">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabDownloader">
      <attribute name="title">
       <string/>
      </attribute>
      <layout class="QFormLayout" name="formLayout_3">
       <property name="horizontalSpacing">
        <number>8</number>
       </property>
       <property name="verticalSpacing">
        <number>8</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="lblVerboseLogging"/>
       </item>
       <item row="0" column="1">
        <widget class="QCheckBox" name="chkVerboseLogging"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lblSponsorBlock"/>
       </item>
       <item row="1" column="1">
        <widget class="QCheckBox" name="chkSponsorBlock"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lblSpeedLimit"/>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="numSpeedLimit">
         <property name="minimum">
          <number>512</number>
         </property>
         <property name="maximum">
          <number>10240</number>
         </property>
         <property name="singleStep">
          <number>512</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lblProxyUrl"/>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="txtProxyUrl"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="lblCookiesBrowser"/>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="cmbCookiesBrowser"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="lblCookiesFile"/>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="txtCookiesFile">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QPushButton" name="btnSelectCookiesFile">
         <property name="icon">
          <iconset theme="document-open"/>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QPushButton" name="btnClearCookiesFile">
         <property name="icon">
          <iconset theme="edit-clear"/>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabConverter">
      <attribute name="title">
       <string/>
      </attribute>
      <layout class="QFormLayout" name="formLayout_4">
       <property name="horizontalSpacing">
        <number>8</number>
       </property>
       <property name="verticalSpacing">
        <number>8</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="lblEmbedMetadata"/>
       </item>
       <item row="0" column="1">
        <widget class="QCheckBox" name="chkEmbedMetadata"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lblRemoveSourceData"/>
       </item>
       <item row="1" column="1">
        <widget class="QCheckBox" name="chkRemoveSourceData"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lblEmbedThumbnails"/>
       </item>
       <item row="2" column="1">
        <widget class="QCheckBox" name="chkEmbedThumbnails"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lblCropAudioThumbnails"/>
       </item>
       <item row="3" column="1">
        <widget class="QCheckBox" name="chkCropAudioThumbnails"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="lblEmbedChapters"/>
       </item>
       <item row="4" column="1">
        <widget class="QCheckBox" name="chkEmbedChapters"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="lblEmbedSubtitles"/>
       </item>
       <item row="5" column="1">
        <widget class="QCheckBox" name="chkEmbedSubtitles"/>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="lblPostprocessingThreads"/>
       </item>
       <item row="6" column="1">
        <widget class="QComboBox" name="cmbPostprocessingThreads"/>
       </item>
       <item row="7" column="1">
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabAria">
      <attribute name="title">
       <string/>
      </attribute>
      <layout class="QFormLayout" name="formLayout_5">
       <item row="0" column="0">
        <widget class="QLabel" name="lblUseAria"/>
       </item>
       <item row="0" column="1">
        <widget class="QCheckBox" name="chkUseAria"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lblAriaMaxConnectionsPerServer"/>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="numAriaMaxConnectionsPerServer">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>16</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lblAriaMinSplitSize"/>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="numAriaMinSplitSize">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>1024</number>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
